testflight/suite_test.go

```diff
@@ -56,15 +58,15 @@ func TestTestflight(t *testing.T) {
 }
 
 var _ = SynchronizedBeforeSuite(func() []byte {
-	var err error
-	config.FlyBin, err = gexec.Build("github.com/concourse/concourse/fly")
```

Comment:
Now that we aren't using `gexec.Build`, this cleanup step doesn't remove the downloaded binary: https://github.com/concourse/concourse/blob/a87b843f471ad81e0d635169c02469840a6dc4e7/testflight/suite_test.go#L112-L115

We should remove `config.FlyBin` after the suite.